api/src/lib/orders/migrations/20211106211249_add_orders_table.js

```diff
@@ -0,0 +1,21 @@
+exports.up = async function (knex) {
+  const sql = `CREATE TABLE orders (
+      id UUID PRIMARY KEY NOT NULL,
+      userid Text NOT NULL,
+      productids Text NOT NULL,
+      totalprice DECIMAL NOT NULL,
+      purchasedate TEXT NOT NULL
+  );
+`
```

Comment:
repeating the #picky comment about using snake_case and appropriate datatypes...